dokka-subprojects/plugin-base/src/test/kotlin/markdown/LinkTest.kt

```diff
@@ -1099,6 +1099,78 @@ class LinkTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun `KDoc link should be unresolved to non-existed property with the name of Kotlin getter #3681`() {
+        testInline(
+            """
+            |/src/main/kotlin/Testing.kt
+            |fun getProperty() = 0
+            |/**
+            |* [property] is unresolved
+            | */
+            | usage() = 0
```

Comment:
shouldn't we have `fun usage()` ?
curious why the test is successful

Reply:
We should. See the fix.